demo/components/filter-component.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
```

Comment:
I won't add this file to pr. The idea is let developers to override filterComponent.